src/components/Counter.js

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import {ButtonCounter} from './ButtonCounter'
+import { Box,Badge,Button} from '@mui/material';
+import { counterIcons } from '../utils';
+
+export function Counter(){
+    const [count, setCount]=React.useState(0);
+        return(
+                <div className='Counter'>
+                    <Box sx={{backgroundColor:'hsl(223, 64%, 98%)'}}>
+                    <ButtonCounter arialabel='Minus' operation={counterIcons.Minus}
+                    onClick={()=>{
+                        setCount(Math.max(count-1,0));
```

Comment:
can you move this function to a handler? the same pattern that is applied in `Home.js` to keep a code standard.